dev-util/gammaray/gammaray-2.8.1.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+QT_MINIMAL="5.9"
+
+DESCRIPTION="High-level runtime introspection tool for Qt applications"
+HOMEPAGE="https://github.com/KDAB/GammaRay \
+		https://www.kdab.com/?s=gammaray"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/GammaRay.git"
+else
+	SRC_URI="https://github.com/KDAB/GammaRay/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~x86 ~amd64"
```

Comment:
Did you actually test on `~x86`? If you did - great! If you didn't, please remove the keyword

Reply:
No, I did not test on x86 but I will remove the keyword.